code/src/components/Album.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Icons from './Icons';
+import Artists from './Artists';
+
+const Album = (props) => {
+
+    return (
+      <article className="album-card">
+        
+      <div className="image-container">
+        <a
+            href={props.item.external_urls.spotify}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+        <Icons />
+        
+        <img src={props.item.images[0].url} alt="album cover" />
+        </a>
+
+      </div>
+
+      <div className="text-wrapper">
+
+        <h2 className="song-title">
+          {props.item.name}
+        </h2>
```

Comment:
Maybe you've could have included this part into the URL container above, to be able to access the album also if clicking on the album name.